### DISCUSSION TITLE
Apache Kafka Connectors

### DISCUSSION THREAD

**Discussion by iamaccosta** [apache/plc4x#1508] [General]
The integrations with Apache Kafka were removed? The links for Source and Sink Connectors in the [apache-kafka] (https://plc4x.apache.org/users/integrations/apache-kafka.html) page do not end at the configuration files.

**Comment by ottlukas** [comment]
The integrations and examples have been moved to: https://github.com/apache/plc4x-extras .

So the link you are looking for is: https://github.com/apache/plc4x-extras/tree/main/plc4j/integrations/apache-kafka/config and yes we will update the links on the website. Thank you for your comment. :-)
